### ISSUE TITLE
[Components and pattern standards] Design components or patterns don't align with Design System guidelines. (04.07.1)

### ISSUE THREAD

**Issue by shiragoodman** [department-of-veterans-affairs/va.gov-team#43143]
#### VFS team name
Public Websites

#### VFS product name
Sitwwide Header

#### VFS feature name
IE11 Deprecation Alert

#### Point of Contact/Reviewers
Sean Sewell (@seansewell) - Design

*For more information on how to interpret this ticket, please refer to the [Anatomy of a Staging Review issue ticket](https://depo-platform-documentation.scrollhelp.site/collaboration-cycle/Anatomy-of-a-Staging-Review-Issue-ticket.2060320997.html) guidance on Platform Website.

---

### Platform Issue
Design components or patterns don't align with Design System guidelines.

### Issue Details
Banner alert background appears to be blue. It should be consistent with component in design system which is grey (#f1f1f1).  

### Link, screenshot or steps to recreate
https://d.pr/i/QAbrzM

https://design.va.gov/components/banner

### VA.gov Experience Standard
[Category Number 04, Issue Number 07](https://depo-platform-documentation.scrollhelp.site/collaboration-cycle/VA.gov-experience-standards.1683980311.html)

### Other References
WCAG SC 3.2.4 AA

---

### Platform Recommendation
Update background color to grey (#f1f1f1).

### VFS Team Tasks to Complete

- [ ] Comment on the ticket if there are questions or concerns
- [ ] Close the ticket when the issue has been resolved or validated by your Product Owner. If a team has additional questions or needs Platform help validating the issue, please comment in the ticket.

**Comment by wesrowe** [comment]
This issue appears only in IE browsers, and appears to be an issue with how color codes are interpreted by IE. We can't address. Closing.
